src/components/LayoutTree.tsx

```diff
@@ -654,9 +654,14 @@ const LayoutTree = () => {
             // Perform this check here instead of in virtualThoughtsPositioned since it changes with the scroll position (though currently `sizes` will change as new thoughts are rendered, causing virtualThoughtsPositioned to re-render anyway).
             if (belowCursor && !isCursor && y > viewportBottom + height) return null
 
+            const parentPath = parentOf(path)
+            const grandParentPath = parentOf(parentOf(path))
+
             return (
               <div
                 aria-label='tree-node'
+                data-parent-path={parentPath}
```

Comment:
Use [hashPath](https://github.com/cybersemics/em/blob/98dff32074ed855a4e9348a2bc00c20cbf3fa42c/src/util/hashPath.ts) for a more explicit conversion of `Path` to a `string`.